packages/erc3k/contracts/IERC3000Executor.sol

```diff
@@ -8,7 +8,12 @@ pragma experimental ABIEncoderV2;
 import "./ERC3000Data.sol";
 
 abstract contract IERC3000Executor {
+    // Note: how is this expected to treat failing actions? especially when having multiple actions but one of them fails
+    // does it worth to allow the user specify whether it should ignore failures or execute in strict mode?
     function exec(ERC3000Data.Action[] memory actions) virtual public returns (bytes[] memory);
 
+    // Note: "actor" sounds a bit confusing, maybe executor, caller, or sender?
+    // Note: not sure how efficient it is to log the whole actions array, is it actually necessary if these are
```

Comment:
After we add a way to link these actions with the Queue events (by allowing to log a random bytes32) we can remove this, since it can be linked off-chain